src/RemoteRequest/CurlRemoteGetRequest.php

```diff
@@ -97,13 +98,17 @@ public function __construct($sslVerify = true, $timeout = self::DEFAULT_TIMEOUT,
     public function get($url, $headers = [])
     {
         $retriesLeft = $this->retries;
+
+        if (! is_string($url) || empty($url)) {
+            throw FailedToGetFromRemoteUrl::withException($url, FailedToParseUrl::forUrl($url));
+        }
+
         do {
             $curlHandle = curl_init();
 
             curl_setopt($curlHandle, CURLOPT_URL, $url);
-            curl_setopt($curlHandle, CURLOPT_HEADER, 0);
```

Comment:
Would passing `false` here also work? I tried looking for any case where `CURLOPT_HEADER` would be enabled by default, but I couldn't. Still, it seems perhaps passing `false` would be a bit safer to be sure?

Reply:
It's false by default so doesn't really matter. It only shows headers when it's passed as `true`. But we can keep it with `false` for visibility.

![image](https://user-images.githubusercontent.com/54371619/196850271-b5683bb1-08df-4bcd-9283-62e82dd84820.png)